src/style.css

```
@@ -1,3 +1,88 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0;
+}
+
+@media screen and (max-width: 1200px) {
+  .header {
+    width: 1200px;
+  }
+}
+
+@media screen and (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+}
+
+.img__logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+  margin-left: 50px;
```

```suggestion
  margin-left: 50px;
```
vertical margins redundant. set horizontal margin 50px for header instead its children